BHoM_UI/Components/oM/CreateCustom.cs

```diff
@@ -168,103 +126,23 @@ public override bool SetItem(object item)
             {
                 Name = ForcedType.Name;
                 Description = ForcedType.Description();
-                InputParams.AddRange(ForcedType.GetProperties().Select(x => GetParam(x)).ToList());
+                InputParams.AddRange(ForcedType.GetProperties().Select(x => x.ToBHoM()).ToList());
             }
             return true;
         }
 
         /*************************************/
 
-        public override string Write()
```

Comment:
Now all components serialise also `InputParams` and `OutputParams` - moved to the general `Caller`.